### ISSUE TITLE
GPG error no pubkey on Raspberry pi 5

### ISSUE THREAD

**Issue by wirytiox** [moonlight-stream/moonlight-qt#1243]
(i did ask for help in discord but i didn't receive any so i post it here, sorry if is not appropriate)
**Describe the bug**
i can't install moonlight-qt in my raspberry pi 5.

**Steps to reproduce**
fresg raspberry pi os install (bookworm) adjust mouse poling to 0 (fastest so is actually usable with moonlight) then run curl -1sLf 'https://dl.cloudsmith.io/public/moonlight-game-streaming/moonlight-qt/setup.deb.sh' | distro=raspbian sudo -E bash

i also installed and configured a wireguard client vpn before installing moonlight, i also did sudo apt-get update and sudo apt-get upgrade

```
curl -1sLf 'https://dl.cloudsmith.io/public/moonlight-game-streaming/moonlight-qt/setup.deb.sh' | distro=raspbian sudo -E bash
Executing the  setup script for the 'moonlight-game-streaming/moonlight-qt' repository ...

   OK: Checking for required executable 'curl' ...
   OK: Checking for required executable 'apt-get' ...
   OK: Detecting your OS distribution and release using system methods ...
 ^^^^: ... Detected/provided for your OS/distribution, version and architecture:
 >>>>:
 >>>>: ... distro=raspbian  version=12  codename=bookworm  arch=aarch64  
 >>>>:
   OK: Checking for apt dependency 'apt-transport-https' ...
   OK: Checking for apt dependency 'ca-certificates' ...
   OK: Checking for apt dependency 'gnupg' ...
   OK: Checking for apt signed-by key support ...
   OK: Importing 'moonlight-game-streaming/moonlight-qt' repository GPG keys ...
   OK: Checking if upstream install config is OK ...
   OK: Installing 'moonlight-game-streaming/moonlight-qt' repository via apt ...
 NOPE: Updating apt repository metadata cache ...
Failed to update via apt-get update
Hit:1 http://deb.debian.org/debian bookworm InRelease
Hit:2 http://deb.debian.org/debian-security bookworm-security InRelease
Hit:3 http://deb.debian.org/debian bookworm-updates InRelease
Get:4 https://dl.cloudsmith.io/public/moonlight-game-streaming/moonlight-qt/deb/raspbian bookworm InRelease [5,122 B]
Err:4 https://dl.cloudsmith.io/public/moonlight-game-streaming/moonlight-qt/deb/raspbian bookworm InRelease
  The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 2F6AE14E1C660D44
Hit:5 http://archive.raspberrypi.com/debian bookworm InRelease
Reading package lists...
W: GPG error: https://dl.cloudsmith.io/public/moonlight-game-streaming/moonlight-qt/deb/raspbian bookworm InRelease: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 2F6AE14E1C660D44
E: The repository 'https://dl.cloudsmith.io/public/moonlight-game-streaming/moonlight-qt/deb/raspbian bookworm InRelease' is not signed.
 ^^^^: ... Failed to update via apt-get update - Context above (maybe no packages?).
 !!!!: Oh no, your setup failed! :-( ... But we might be able to help. :-)
 !!!!: 
 !!!!: You can contact Moonlight Game Streaming for further assistance.
 !!!!: 
 !!!!: URL: https://github.com/moonlight-stream/moonlight-qt
 !!!!: 

```

**Comment by cgutman** [comment]
Probably a transient Cloudsmith issue. Let me know if it occurs again.

**Comment by wirytiox** [comment]
> Probably a transient Cloudsmith issue. Let me know if it occurs again.

hello, i uninstalled moonlight and tried reinstalling and got the same issue. `~ $ curl -1sLf 'https://dl.cloudsmith.io/public/moonlight-game-streaming/moonlight-qt/setup.deb.sh' | distro=raspbian codename=$(lsb_release -cs) sudo -E bash
Executing the  setup script for the 'moonlight-game-streaming/moonlight-qt' repository ...

   OK: Checking for required executable 'curl' ...
   OK: Checking for required executable 'apt-get' ...
   OK: Checking for apt dependency 'apt-transport-https' ...
   OK: Checking for apt dependency 'ca-certificates' ...
   OK: Checking for apt dependency 'gnupg' ...
   OK: Checking for apt signed-by key support ...
   OK: Importing 'moonlight-game-streaming/moonlight-qt' repository GPG keys ...
   OK: Checking if upstream install config is OK ...
   OK: Installing 'moonlight-game-streaming/moonlight-qt' repository via apt ...
 NOPE: Updating apt repository metadata cache ...
Failed to update via apt-get update
Hit:1 http://deb.debian.org/debian bookworm InRelease
Hit:2 http://deb.debian.org/debian-security bookworm-security InRelease
Hit:3 http://deb.debian.org/debian bookworm-updates InRelease
Hit:4 https://brave-browser-apt-release.s3.brave.com stable InRelease
Hit:5 http://archive.raspberrypi.com/debian bookworm InRelease
Get:6 https://dl.cloudsmith.io/public/moonlight-game-streaming/moonlight-qt/deb/raspbian bookworm InRelease [5,122 B]
Err:6 https://dl.cloudsmith.io/public/moonlight-game-streaming/moonlight-qt/deb/raspbian bookworm InRelease
  The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 2F6AE14E1C660D44
Reading package lists...
W: GPG error: https://dl.cloudsmith.io/public/moonlight-game-streaming/moonlight-qt/deb/raspbian bookworm InRelease: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 2F6AE14E1C660D44
E: The repository 'https://dl.cloudsmith.io/public/moonlight-game-streaming/moonlight-qt/deb/raspbian bookworm InRelease' is not signed.
 ^^^^: ... Failed to update via apt-get update - Context above (maybe no packages?).
 !!!!: Oh no, your setup failed! :-( ... But we might be able to help. :-)
 !!!!: 
 !!!!: You can contact Moonlight Game Streaming for further assistance.
 !!!!: 
 !!!!: URL: https://github.com/moonlight-stream/moonlight-qt
 !!!!: 
`

**Comment by wirytiox** [comment]
anyone that runs into this issue i couldn't find a fix, you'll need to do a eManuel install:
i went into: https://cloudsmith.io/~moonlight-game-streaming/packages/?q=moonlight-qt and downloaded the appropiate version (basically the arm64 debian/bookworm and latest) and installed it like any other package (if any of you arent tech savvy just ask chatgpt how to install a .deb package)
